crates/ruff_linter/resources/test/fixtures/pyflakes/F401_29__all_pluseq/__init__.py

```diff
@@ -0,0 +1,18 @@
+"""__init__.py with __all__ populated by plus-eq
+
+multiple __all__ so cannot offer a fix to add to them; offer fixes for redundant-aliases
+"""
+
+
+from . import exported  # Ok: is exported in __all__
+
+
+from . import unused # F401: convert to redundant-alias (with fix)
+
+
+from . import renamed as bees # F401: convert to redundant-alias (no fix)
+
+
+__all__ = []
+__all__ += ["exported"]
```

Comment:
I'm not sure there's much value in a fixture like this, as it doesn't seem particularly realistic to me. In real code, you'd only have multiple `__all__` definitions at the top level if there was some kind of version or platform difference that made an attribute sometimes available, sometimes not (and as soon as you have that kind of branching, attempting an autofix becomes much more complex, as we've discussed!). So I think the example you added in `crates/ruff_linter/resources/test/fixtures/pyflakes/F401_30__all_conditional/__init__.py` is much better; I'd remove this example, I think.

Reply:
Ok, I'll remove it